bootstrap.sh

```diff
@@ -401,6 +401,7 @@ case "$cmd" in
     export CI=1
     export CI_NIGHTLY=1
     build
+    release-image/bootstrap.sh push
```

Comment:
This pushes the image tagged with the current commit in order for GKE to be able to use it